packaging/aleph-vm/etc/systemd/system/ipfs.service

```
@@ -76,7 +76,9 @@ Type=notify
 User=ipfs
 Group=ipfs
 Environment=IPFS_PATH="/var/lib/ipfs"
-ExecStart=/opt/kubo/ipfs daemon --init --migrate --init-profile=server --config-file /etc/ipfs/kubo.json
+ExecStartPre=/opt/kubo/ipfs init
+ExecStartPre=/opt/kubo/ipfs config --json Gateway.PublicGateways '{"localhost": {"UseSubdomains": false, "Paths": ["/ipfs", "/ipns"]}}'
+ExecStart=/opt/kubo/ipfs daemon --init --migrate=true --init-profile=server
```

Remove redundant `init`
```suggestion
ExecStart=/opt/kubo/ipfs daemon --migrate=true --init-profile=server
```